easy-note/cmd/user/dal/db/init.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package db
+
+import (
+	"easy-note/pkg/consts"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+	"gorm.io/plugin/opentelemetry/tracing"
+)
+
+var DB *gorm.DB
+
+// Init init DB
+func Init() {
+	var err error
+	DB, err = gorm.Open(mysql.Open(consts.MySQLDefaultDSN),
+		&gorm.Config{
+			PrepareStmt: true,
+			Logger:      logger.Default.LogMode(logger.Info),
```

Comment:
use https://github.com/go-gorm/opentelemetry#set-logger
How the logs of the three frames are linked.